os/arch/arm/src/armv7-a/arm_cpupause.c

```diff
@@ -417,4 +417,53 @@ int up_cpu_resume(int cpu)
 	return OK;
 }
 
+/****************************************************************************
+ * Name: up_cpu_pause_all
+ *
+ * Description:
+ *   pause all the CPUs other than the current cpu. Internally, it calls 
+ * 	 up_cpu_pause() api to stop all the CPU's.
+ *
+ * Returned Value:
+ *   Zero on success; a negated errno value on failure.
```

Comment:
It always returns OK. Need the return?

Reply:
I think we make return type void.

Reply:
You changed the return type to void, but this line is not changed.